db/migrate/20230223225921_add_user_to_reviews.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToReviews < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :reviews, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
Instead of making a migration file to add the user can you edit the initial migration file instead ?
Just add `t.references :user, null: false, foreign_key: true` to the CreateReviews migration then drop your db, create it again and migrate it